yahooquery/ticker.py

```diff
@@ -1330,9 +1330,9 @@ def _historical_data_to_dataframe(self, data, params, adj_timezone):
             df = pd.DataFrame(columns=["high", "low", "volume", "open", "close"])
         else:
             if "dividends" in df.columns:
-                df["dividends"].fillna(0, inplace=True)
+                df.fillna({"dividends": 0}, inplace=True)
             if "splits" in df.columns:
-                df["splits"].fillna(0, inplace=True)
+                df.fillna({"splites": 0}, inplace=True)
```

Comment:
Hi, @maread99 ! There seems to be a typo in the word "splits".
P.S.: I'm not an actual reviewer, just passing by

Reply:
Crumbs, just as well you did pass by. Thanks!

Force-pushed a fix.

Reply:
There seem to be some other areas with the same warning now as well